docs: new intro page (#1384)
edgehog-device-manager/edgehog: backend/docs/pages/intro/intro.md

<h3>OTA update campaigns</h3>
<p>Create base image collections and roll out firmware upgrades across device groups using campaigns.</p>
</div>

<div class="feature-card">
<div class="card-icon" style="background: #e8f0ff;">📦</div>
<h3>Container deployments</h3>
<p>Define applications as versioned releases of containers. Deploy, upgrade, start, stop, and delete containerized workloads via deployment campaigns.</p>
</div>

<div class="feature-card">
<div class="card-icon" style="background: #fce8f0;">📊</div>
<h3>Fleet grouping</h3>
<p>Tag devices freely and write selector expressions to form dynamic groups that update automatically.</p>
</div>

<div class="feature-card">
<div class="card-icon" style="background: #eaf3de;">📍</div>
<h3>Geolocation</h3>
<p>Track device position and location data reported from the field. Query per-device position and last-seen coordinates through the API.</p>
</div>

<div class="feature-card">
<div class="card-icon" style="background: #eeedfe;">🔗</div>
<h3>Remote Forwarding</h3>
<p>Request and manage forwarder sessions to tunnel traffic to specific devices for diagnostics and remote access — configurable per tenant.</p>
</div>
</div>

## API reference

> ### Admin Rest APIs

Reviewer: ```suggestion
### Admin REST APIs
```

> Edgehog admin tasks (i.e., managing tenant) are available trough a Rest API. It support creating, updating and deleting tenants

Reviewer 2: ```suggestion
Edgehog admin tasks (i.e., managing tenants) are available trough a Rest API. It supports creating, updating and deleting tenants.
```

> Edgehog admin tasks (i.e., managing tenants) are available trough a Rest API. It supports creating, updating and deleting tenants

Reviewer: ```suggestion
Edgehog admin tasks (i.e., managing tenants) are available through a REST API. It supports creating, updating and deleting tenants
```

**Endpoint**

```
http://<your-host>/admin-api/v1/
```

Replace `<your-host>` with your Edgehog instance hostname.

> **Full API docs** available here: [admin-rest-api](admin-rest-api/)

Reviewer: The link to the Admin REST API doesnt work

Author: These link work when creating the documentation with `mix edgehog.docs`, since that command generates the correct foldrer. Same goes for `tenant-graphql-api`)

### Tenant GraphQL APIs

All Edgehog operations are exposed through a **GraphQL API**. It supports queries, mutations, and real-time subscriptions.

**Endpoint**

```
http://<your-host>/tenants/<your-tenant>/api
```

Replace `<your-host>` with your Edgehog instance hostname and `<your-tenant>` with the slug of your tenant.

> **Full API docs** available here: [tenant-graphql-api](tenant-graphql-api/)

Reviewer: Same with the Tenant GraphQL API

#### Key resources

| [**Queries**](https://docs.edgehog.io/0.12/tenant-graphql-api/#group-Operations-Queries) | Query your device data. |

> | [**Mutations**](https://docs.edgehog.io/0.12/tenant-graphql-api/#group-Operations-Mutations) | Make changes and interact with edgehog. |

Reviewer: ```suggestion
| [**Mutations**](https://docs.edgehog.io/0.12/tenant-graphql-api/#group-Operations-Mutations) | Make changes and interact with Edgehog. |
```

| [**Subscriptions**](https://docs.edgehog.io/0.12/tenant-graphql-api/#group-Operations-Subscriptions) | Subscribe to events as they happen. |

## How to read this documentation

> **1. Core concepts**

Reviewer: Remove all `—` and replace them with commas or other separators where needed.
